pkg/runtime/runtime.go

```diff
@@ -962,6 +962,7 @@ func (a *DaprRuntime) initActors(ctx context.Context) error {
 		AppID:             a.runtimeConfig.id,
 		ActorsService:     a.runtimeConfig.actorsService,
 		RemindersService:  a.runtimeConfig.remindersService,
+		SchedulerService:  os.Getenv("SCHEDULER_ADDRESS"),
```

Comment:
```suggestion
		SchedulerService:  os.Getenv("DAPR_SCHEDULER_ADDRESS"),
```

Reply:
I can change this but it is a hack until we pass it correctly from cmd args.